app/jobs/get_next_ai_message_job.rb

```diff
@@ -0,0 +1,45 @@
+class GetNextAiMessageJob < ApplicationJob
+  def perform(conversation_id)
+    puts "GetNextAiMessageJob.perform(#{conversation_id})"
```

Comment:
I'd normally leave this kind of thing up to the queue to do, but it's really terse in it's output right now. I found it helpful to have this in the logs so that I could see that _something_ was happening.